packages/backend/src/graphql/resolvers/page.test.ts

```diff
@@ -52,12 +52,18 @@ export const getPageTests = <T extends Model, M extends BaseModel>({
   const toConnection = (query: ApolloQueryResult<T>): Connection<T> => {
     if (query.data) {
       if (parent) {
+        // eslint-disable-next-line @typescript-eslint/ban-ts-comment
```

Comment:
there is an exact test suite but in the `auth` package instead, not sure why it is not complaining there at first glance